src/main/java/UseCase/CreatePlaylistInteractor.java

```diff
@@ -0,0 +1,49 @@
+package UseCase;
+import Database.playlistDsData;
+import Entities.Playlist;
+import InputBoundary.newPlaylistInputBoundary;
+import InputData.newPlaylistInputData;
+import Database.playlistAccessInterface;
+import Database.playlistLibrary;
+import OutputBoundary.newPlaylistOutputBoundary;
+import OutputData.newPlaylistOutputData;
+
+public class CreatePlaylistInteractor implements newPlaylistInputBoundary {
+        private newPlaylistOutputBoundary outputBoundary;
+//        private String message;
+    public CreatePlaylistInteractor(newPlaylistOutputBoundary outputBoundary){
+        this.outputBoundary = outputBoundary;
+//        this.message = "";
+    }
+    /**
+     * @param newplaylistInputData holds necessary data to instantiate a new playlist
+     * if hasFirstSong() is false, then newPlaylist() will instantiate an empty playlist and vice versa if it's value
+     * is true
+     */
+    public void newPlaylist(newPlaylistInputData newplaylistInputData) {
+        Playlist playlist;
+        playlistAccessInterface library = playlistLibrary.getInstance();
```

Comment:
(Mostly for consistency) Add a `private final playlistAccessInterface library;` up top and move line 25 to the initializer